dags/monrecap.py

```diff
@@ -154,7 +130,8 @@ def monrecap_gsheet(**kwargs):
 
     (
         create_schema(DB_SCHEMA).as_setup()
-        >> [monrecap_airtable(), monrecap_gsheet()]
+        >> create_models(MonrecapBase)
```

Comment:
```suggestion
        >> create_models(MonrecapBase).as_setup()
```
Je pense même que tu peux supprimer la tâche `create_schema` pour ne garder que la `create_models`, celle-ci créant le _schema_ automatiquement.